app/Models/Post.php

```diff
@@ -55,6 +56,15 @@ public function likes()
         return $this->hasMany('App\Models\Like', 'post_id');
     }
 
+    /**
+     * Lay bai post duoc like thoe user
+     */
+    public function liked($post_id) {
+        $liked = $this->likes()->where('post_id', $post_id)->where('user_id', Auth::id())->first();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Models/Post.php</br>**Line**: 63</br>Variable "post_id" is not in valid camel caps format